aws-proofs/run

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Copyright 2021, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+
+# ./run [ci-actions-branch]
+#
+# This script is the entry point on the AWS instance.
+#
+# Currently, the VM image must be updated manually if this script changes.
```

Comment:
Am I right in understanding that you've built an AMI that's just some base distro plus some pre-configuration to make it  suitable for running tests? (e.g. a `test-runner` user, this script, plus some things installed). Is the process of building the AMI documented somewhere? It sounds like you intend to automate building the AMI, but that's still on the TODO list. (It's not a problem, I'm just curious.)